index.html

```diff
@@ -8,33 +6,43 @@
 <title>
 Lawrence's CTD portfolio
 </title>
-
   <meta charset="UTF-8">
   <meta name="description" content="CTD portfolio">
   <meta name="keywords" content="HTML, CSS, JavaScript">
   <meta name="author" content="Lawrence">
 </head>
 
 <body>
-Lawrence
+  <p>
+    Lawrence
+  </p>
 
 <section>
 <h1>About</h1>
 <p>I am learning Javascript so that I can learn to make React apps.</p>
+</section>
 
 <section>
 <h1>Experience</h1>
 <p>I am currently attending Code the Dream bootcamp.</p>
+</section>
+
+<section id="skills">
+<h2>Skills</h2>
+<ul>
+</ul>
+</section>
 
 <section>
 <h1>Connect</h1>
-<p>
 <ul>
-<li>Email: example@example.com
-<li>Github: https://github.com/LawrenceELee
-<li>LinkedIn: none
-<li>Twitter: none
+  <li>Email: example@example.com</li>
+  <li>Github: https://github.com/LawrenceELee</li>
+  <li>LinkedIn: none</li>
+  <li>Twitter: none</li>
 </ul>
-</p>
-</body>
-</html>
```

Comment:
While you _technically_ can omit the closing body and html tags (and many other closing tags depending on circumstance), I personally think it's clearer to include them.